src/validations/rules/ssp.sch

```diff
@@ -4585,7 +4625,8 @@
         <sch:diagnostic
             doc:assertion="party-has-one-responsibility"
             doc:context="oscal:party[@type eq 'person']"
-            id="party-has-one-responsibility-diagnostic"><xsl:value-of select="o:name"/> - This person has more than one responsibility.</sch:diagnostic>
+            id="party-has-one-responsibility-diagnostic"><xsl:value-of
+                select="o:name" /> - This person has more than one responsibility.</sch:diagnostic>
```

Comment:
This section of code is not related to the issue. Why was it changed?

Reply:
Change in formatting rules in oXygen.  It looks like automatic line length and space before empty tag closure.  There is no change to the code nor the output string.